modules/planning/planning_base.cc

```diff
@@ -260,8 +260,13 @@ Status PlanningBase::Plan(
 
   ADEBUG << "current_time_stamp: " << std::to_string(current_time_stamp);
 
-  last_publishable_trajectory_->PrependTrajectoryPoints(
-      stitching_trajectory.begin(), stitching_trajectory.end() - 1);
+  // Navi Panner doesn't need to stitch the last path planning
```

Comment:
Please use gflag when staring navi planner to control whether to stitch trajectory or not.

Reply:
ok